### ISSUE TITLE
Cannot resolve symbole AGVRecyclerViewAdapter 

### ISSUE THREAD

**Issue by MahmoudB** [felipecsl/AsymmetricGridView#57]
As the title of the issue shows AGVRecyclerViewAdapter cannot be resolved, when I checked the decompiled classes in External Libraries section on my Android Studio project I didn't find it there.

I saw the AGVRecyclerViewAdapter in the source code of the project in Github, is there a difference between the code base and the gradle dependency ?

**Comment by mikigom** [comment]
I got the same issue, too...

**Comment by jordantakor** [comment]
Yes the gradle version in the readme and current master branch is different. You should use the snapshot available on sonatype. 

Here is how to add it in gradle

First add sonatype as a repository
```
repositories {
    maven { url("https://oss.sonatype.org/content/repositories/snapshots/") }
}
```

Then add the snapshot dependancy
```
dependancies {
     compile 'com.felipecsl.asymmetricgridview:library:3.0.0-SNAPSHOT'
}
```
